src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultProjectSnapshot.cs

```diff
@@ -59,7 +59,7 @@ public override DocumentSnapshot GetDocument(string filePath)
                     _documents.Add(filePath, result);
                 }
 
-                return result;
+                return result!;
```

Comment:
😢 Sadly, this was the best thing that I could do for now. It's entirely possible that this can return `null`. It seems like most code that uses this API assumes that it won't return `null`, but tests fail if I add `Assumes.NotNull(result)`.

Reply:
Worth an issue for follow up?

Reply:
Yup! I'll file one before I move this PR out of draft.

Reply:
Filed #7945